src/shims/x86/mod.rs

```diff
@@ -1144,51 +1155,55 @@ fn pclmulqdq<'tcx>(
     right: &OpTy<'tcx>,
     imm8: &OpTy<'tcx>,
     dest: &MPlaceTy<'tcx>,
+    width: u64,
```

Comment:
The doc comment should explain the meaning of this argument

Reply:
Fixed. I also renamed it to `len` since we're usually talking about vecor _length_.